clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -460,37 +463,63 @@ async function fetchCustomFidesCss(
   if (shouldRefresh) {
     try {
       const fidesUrl = getFidesApiUrl();
-      const response = await fetch(
-        `${fidesUrl}/plus/custom-asset/custom-fides.css`,
+      const css = await pRetry(
+        async () => {
+          const assetResponse = await fetch(
+            `${fidesUrl}/plus/custom-asset/custom-fides.css`,
+          );
+          if (assetResponse.status === 404) {
+            log.debug(
+              "No custom-fides.css found, disabling Custom CSS polling.",
+            );
+            autoRefresh = false;
+            return null;
+          }
+
+          const data = await assetResponse.text();
+          if (!data) {
+            throw new Error("No data returned by the server");
+          }
+          if (!assetResponse.ok) {
+            log.error(
+              "Error fetching custom-fides.css:",
+              assetResponse.status,
+              assetResponse.statusText,
+              data,
+            );
+            throw new Error(
+              `HTTP error occurred. Status: ${assetResponse.status}`,
+            );
+          }
+
+          return data;
+        },
+        {
+          retries: CUSTOM_CSS_MAX_RETRIES,
+          factor: CUSTOM_CSS_BACKOFF_FACTOR,
+          minTimeout: CUSTOM_CSS_RETRY_MIN_TIMEOUT_MS,
+          onFailedAttempt: (error) => {
+            log.debug(
```

Comment:
Related to above nit - I wasn't able to see my debug logs but if this happens on each attempt its probably sufficient. If it only happens at the end (like the built in pRetry error I am seeing), then it would always show 0 for attempts remaining

This might all be my setup not quite right as well

Reply:
Discussed in Slack & after configuring debug logging this works as expected.